apisix/plugins/limit-req.lua

```diff
@@ -102,10 +102,9 @@ function _M.access(conf, ctx)
     end
 
     if key == nil then
-        core.log.info("bypass the limit req as the key is empty")
-        -- Bypass the limit req when the key is empty.
-        -- This behavior is the same as Nginx
-        return
+        core.log.info("The key of limit req is empty or invalid, set $remote_addr as the key")
```

Comment:
```suggestion
        core.log.info("The value of the configured key is empty, use client IP instead")
```

The value is empty, not the key itself.

Reply:
Done, thanks.